src/Microsoft.AspNet.Identity.EntityFramework/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
```

Comment:
All the properties now have public setters. I'll change these to public as well then